docker/run_worker_queue.sh

```diff
@@ -10,10 +10,17 @@ if [ -z "$REDIS_PASSWORD" ]; then
     exit 1
 fi
 
+# Make bash more strict about errors
+set -euo pipefail
```

Comment:
Does this work before something using `test`? I recall having tripped on `set -e` making all `if`s fail.

Reply:
I'm not aware if `test` and `set -e` in general might interfere, but at least here it works. Tested.